pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -195,6 +196,14 @@ func convertProtocolHTTP(http *v1beta.HTTPProtocol) string {
 	return strings.Join(keywords, " ")
 }
 
+func convertProtocolTLS(tls *v1beta.TLSProtocol) string {
+	var keywords []string
+	if tls.SNI != "" {
+		keywords = append(keywords, fmt.Sprintf(`tls.sni; content:"%s"; nocase; pcre:"/%s$/;"`, tls.SNI, tls.SNI))
+	}
+	return strings.Join(keywords, " ")
```

Comment:
Do you need a slice `keywords` here? seems there will be only one string.

Reply:
We could add more key word in the future.